cornflow/commands/commands.py

```diff
@@ -144,11 +227,32 @@ def run(self):
         db.session.bulk_save_objects(views_list)
         db.session.commit()
 
+        if verbose == 1:
+            print("Endpoints successfully registered")
+
         return True
 
 
 class UpdateViews(Command):
-    def run(self):
+    def get_options(self):
+        return (
+            Option(
+                "-v",
+                "--verbose",
+                dest="verbose",
+                help="Verbose for the command. 0 no verbose, 1 full verbose",
+                type=int,
+            ),
```

Comment:
in fact inf you're using this in all functions... couldn't you just create it once at the top of the file such as:
```
verbose_option = Option("-v", ...)
```
and then just pass `verbose_option` everywhere?

Reply:
Will try